modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/entity/Playlist.kt

```diff
@@ -156,4 +150,19 @@ data class Playlist(
 
     val isAllEpisodes: Boolean
         get() = unplayed && partiallyPlayed && finished && notDownloaded && downloaded && audioVideo == AUDIO_VIDEO_FILTER_ALL && allPodcasts && !filterDuration && filterHours == 0 && !starred
+
+    fun sortOrder() = SortOrder.fromInt(sortId)
+
+    enum class SortOrder(val value: Int) {
```

Comment:
I created this enum to minimize how much we have to pass around and match against plain `Int`s.